### PR TITLE
Input parameter list

### PR DESCRIPTION
**Pull request by Bonnarel**
Small change in order to make service auto description  give the list of string PARAMETERS available in the service recommended by a SHOULD.  Solves Issue #1

### CODE REVIEW

File: SIA.tex [hunk 1]
```diff
@@ -506,7 +496,7 @@ \subsubsection{UPLOAD}
  The DALI UPLOAD parameter is not used by this version of SIA. The use case of uploading lists of coordinates is covered by the multiple-valued parameters values.
 
 \subsubsection{Service PARAMETER self description}
-Any service may  include a DataLink service descriptor in the VOTable output to describe itself. This descriptor would describe the supported query parameters (standard and custom), including list of values for those with a fixed list (e.g. COLLECTION, INSTRUMENT, FACILITY, DPTYPE, CALIB, and FORMAT).
+Any service SHOULD  include a DataLink service descriptor in the VOTable output to describe itself. This descriptor would describe the supported query parameters (standard and custom), including list of values for those with a fixed list (COLLECTION, INSTRUMENT, FACILITY, DPTYPE, CALIB, and FORMAT). This is peculiarly important for those parameters where the fixed list is not standardized by any specification such as COLLECTION, INSTRUMENT and FACILITY.
```
Comment (jd-au) [inline SIA.tex]: I think `peculiarly` should be `particularly`

File: ivoatexmeta.tex [hunk 2]
```diff
@@ -0,0 +1,6 @@
+% GENERATED FILE -- edit this in the Makefile
```
Comment (jd-au) [inline ivoatexmeta.tex]: I think this file should be ignored

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+name: Update PDF Preview
```
Comment (jd-au) [inline .github/workflows/main.yml]: Is this file needed?

**Review by jd-au — CHANGES_REQUESTED** [review]

File: SIA.tex (hunk 1 above)
Comment (Bonnarel) [reply SIA.tex]: This change has been made (although I'm not able to distinguish the meaning of the two words)

File: ivoatexmeta.tex (hunk 2 above)
Comment (Bonnarel) [reply ivoatexmeta.tex]: Hum. Without changing ivoatexmeta.tex I don't think the date and draft title can change ? Can it ?

Comment (jd-au) [reply ivoatexmeta.tex]: If this file is not present it is generated by the make command using the metadata in `Makefile` so I'd recommend making these changes there and adding dropping this file.

Comment (Bonnarel) [reply ivoatexmeta.tex]: OK. understood. I fixed that.

File: .github/workflows/main.yml (hunk 3 above)
Comment (Bonnarel) [reply .github/workflows/main.yml]: No. removed

**Review by jd-au — APPROVED** [review]
